src/dmd/doc.d

```diff
@@ -1860,6 +1860,34 @@ extern (C++) const(char)* skipwhitespace(const(char)* p)
     return p;
 }
 
+/************************************************
+ * Scan past all instances of the given characters.
+ * Params:
+ *  buf =           an OutputBuffer containing the DDoc
+ *  i =             the index within `buf` to start scanning from
+ *  chars =         the characters to skip; order is unimportant
+ * Returns: the index after skipping characters.
+ */
+private size_t skipChars(OutBuffer* buf, size_t i, string chars)
+{
+    const slice = buf.peekSlice();
+    for (; i < slice.length; i++)
+    {
+        bool contains = false;
+        foreach (char c; chars)
+        {
+            if (slice[i] == c)
+            {
+                contains = true;
+                break;
+            }
+        }
+        if (!contains)
+            break;
+    }
+    return i;
```

Comment:
Just for fun:
```
Outer:
foreach (j, c; buf.peekSlice()[i .. length])
{
    foreach (d; chars)
    {
        if (d == c)
              continue Outer;
    }
    return i + j;
}
return buf.offset;

Reply:
Eliminating state flags (i.e. `contains`) using control flow generally makes the code easier to understand.

Reply:
Nice—I had no idea a label could work in front of a foreach loop like that. Thanks!